loops-and-arrays.js

```diff
@@ -12,6 +12,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+for (i=0; i<names.length; i++) {
+    names[i] = names[i] + "je";
+}
+console.log(...names);
```

Comment:
Is er een reden dat je de ...  voor names gebruikt in je console.log? Ik ken dit gebruik niet, ook niet zo snel iets kunnen vinden op internet, of doet je idea dit automatisch?